### PR TITLE
Migrate without updating

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@
     t.string "lane_id", default: "default", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.boolean "active_version", default: false
+    t.boolean "active_version"
```
Comment (justinlittman) [inline db/schema.rb]: Why did you remove the default?

Comment (jcoyne) [reply db/schema.rb]: Oops, fixed that.
